ml_dtypes/tests/finfo_test.py

```diff
@@ -71,6 +82,13 @@ def assert_zero(val):
 
     assert_representable(info.tiny)
     assert_representable(info.max)
+    assert_representable(info.min)
+
+    msg = "overflow encountered in nextafter"
+    with ignore_warning(category=RuntimeWarning, message=msg):
+      assert_infinite(np.nextafter(info.max, make_val(np.inf)))
```

Comment:
Isn't it the case that the e4m3 types don't support infinities? Shoudn't this test fail for them?

Reply:
Worth a comment, but SGTM.